version.bzl

```diff
@@ -32,3 +32,13 @@ def check_version(min_required, max_supported):
     max = _parse_version(max_supported)
     if max < _bound_size(found_version, len(max)):
         fail("Your bazel is too new. Maximum supported version {} of bazel, found {}".format(max_supported, found))
+
+def blacklist_version(version, reason):
```

Comment:
In retrospect, it would have been nice to make this a list inside the `check_version` rule so that the error could include a suggestion of which versions are allowed. (It is almost certainly not worth re-doing though)

Reply:
I considered doing it that way, but this seemed simpler.